pkg/graveler/ref/manager.go

```diff
@@ -387,6 +395,31 @@ func (m *Manager) addCommit(tx db.Tx, repositoryID graveler.RepositoryID, commit
 	return err
 }
 
+func (m *Manager) updateCommitGeneration(tx db.Tx, repositoryID graveler.RepositoryID, commitIDToNode map[graveler.CommitID]CommitNode) error {
+	for len(commitIDToNode) != 0 {
+		command := "WITH updated(id, generation) AS (VALUES "
+		var updatingRows int
+		for commitID, commitNode := range commitIDToNode {
+			if updatingRows != 0 {
+				command += ","
+			}
+			command += fmt.Sprintf("('%s', %d)", commitID, commitNode.generation)
```

Comment:
This is SQL injection if `commitID` contains bad characters.  Rather than trust the file, let's quote the string!  Typically the golang DB packages have ways of quoting a literal string.

Reply:
Which string do you think we should quote? the `commitID` field is already quoted. What am I missing? 😅

Reply:
Which string do you mean we should quote? I'm already quoting the `commitID`. Am I'm missing something? 😅